app/user.go

```diff
@@ -149,13 +149,15 @@ func (a *App) CreateUserWithInviteId(user *model.User, inviteId, redirect string
 		return nil, err
 	}
 
+	ruser.DisableWelcomeEmail = user.DisableWelcomeEmail
```

Comment:
Would it make sense to add this to the create user `func`? Preserving the value of a non-persisted property in the create code could be valuable and less error prone down the line 1/5

Reply:
Yep agree, much better.

Reply:
Hey @mgdelacroix , I have made the suggested changes